site/source/user-manual/connecting/connecting-lan/lan-browser/lan-brave.rst

```diff
@@ -8,7 +8,7 @@ Trust Embassy CA in Brave
 
 .. tabs::
 
-    .. group-tab:: Linux / macOS
+    .. group-tab:: Linux
 
         #. Open a new tab in Brave and open to "Settings" from the top-right hamburger menu.
```

Comment:
Maybe note: No additional setup if using system-wide proxy, which we ought to default to going forward

Reply:
I will change this once you have added instructions for linux system-wide